packager/react-packager/src/DependencyResolver/fastfs.js

```diff
@@ -297,7 +297,7 @@ class File {
     Object.keys(this.children).forEach(key => {
       const file = this.children[key];
       if (file.isDir) {
-        files.push(...file.getFiles());
+        file.getFiles().forEach((f) => files.push(f));
```

Comment:
Actually, I just tried `files.push.apply(files, file.getFiles())` and I got `RangeError: Maximum call stack size exceeded` again. I suspect I'm hitting some sort of built-in limit for maximum number of arguments, and it's just surfacing as a stack size exceeded error.

Also, it seems Babel translates `files.push(...file.getFiles());` to  `files.push.apply(files, file.getFiles())`, which is what you'd expect. I was racking my brain to figure out why this syntax was incurring recursion...

Anyway, I'll add a comment nonetheless.

P.S.: It's almost 2016 and JS arrays still suck.